client/src/main/java/com/vesoft/nebula/client/graph/net/SyncConnection.java

```diff
@@ -169,7 +169,11 @@ public ExecutionResponse execute(long sessionID, String stmt)
                     throw new IOErrorException(IOErrorException.E_NO_OPEN, te.getMessage());
                 } else if (te.getType() == TTransportException.TIMED_OUT
                     || te.getMessage().contains("Read timed out")) {
-                    reopen();
+                    try {
+                        reopen();
+                    } catch (ClientServerIncompatibleException ex) {
+                        throw new IOErrorException(IOErrorException.E_TIME_OUT, te.getMessage());
```

Comment:
why error type is E_TIME_OUT?

Reply:
wrong exception var, te -> ex

Reply:
that's the thing i actually wanted to confirm with you, what should the exact type here to be thrown